roles/setup_pgd/defaults/main.yml

```diff
@@ -113,3 +113,10 @@ supported_pg_version:
 
 supported_pgd_version:
   - 5
+
+# PGD Commit Scopes
+configure_pgd_camo: true
+configure_pgd_groupcommit: false
+pgd_camo_scope_name: "pgd_camo_scope"
+pgd_groupcommit_scope_name: "pgd_groupcommit_scope"
+pgd_two_node_group_name: "pgd_two_nodes"
```

Comment:
this would only allow for one `pgd_two_node_group_name` to be input, should we allow for a list? Or make it a list of dictionaries with keys like `commit_scope_type`, `commit_scope_group`, `commit_scope_name` to allow for more than one CAMO or group commit setup?

Reply:
So far architecturally, a bdr node group should only host two nodes.

If we add more support for other commit scopes, it will directly impact the deadlines for these benchmarks.

This is another subject for a larger discussion that can directly impact the deadline of these benchmarks.